rule/rule.go

```diff
@@ -115,24 +130,40 @@ func LoadWorkspaceData(path, pkg string, data []byte) (*File, error) {
 	if err != nil {
 		return nil, err
 	}
-	return ScanAST(pkg, ast), nil
+	return ScanAST(pkg, ast, ast.Stmt), nil
+}
+
+// LoadMacroData is similar to LoadData but parses the data as a
+// bzl file.
+func LoadMacroData(path, pkg, defName string, data []byte) (*File, error) {
+	ast, err := bzl.ParseDefault(path, data)
+	if err != nil {
+		return nil, err
+	}
+	defStmt := scanForDef(ast.Stmt, defName)
+	if defStmt == nil {
+		return nil, fmt.Errorf("Could not find function name in .bzl file %s", defName)
```

Comment:
I don't actually think this should be an error. If the function doesn't already exist, `Sync` should create it.

Reply:
Fixed and tested with TestImportReposFromDepToEmptyMacro